Makefile

```diff
@@ -3,10 +3,10 @@ CHART := java
 RELEASE := chart-${CHART}-release
 NAMESPACE := chart-tests
 TEST := ${RELEASE}-test-service
-ACR := hmctssandbox
-ACR_SUBSCRIPTION := DCD-CFT-Sandbox
-AKS_RESOURCE_GROUP := cnp-aks-sandbox-rg
-AKS_CLUSTER := cnp-aks-sandbox-cluster
+ACR := hmcts
+ACR_SUBSCRIPTION := DCD-CNP-DEV
+AKS_RESOURCE_GROUP := cnp-aks-rg
+AKS_CLUSTER := cnp-aks-cluster
 
```

Comment:
It's probably better not to modify these from the sandbox values

Reply:
Am reverting back to sandbox values ... However, as I mentioned to you today, using the Makefile with the Sandbox doesn't work out of box because of the subscriptionId in ci-values.yaml
thanks